ctapipe/configuration/configuration.py

```diff
@@ -0,0 +1,312 @@
+'''
+https://docs.python.org/2/library/argparse.html
+'''
+
+
+from argparse import ArgumentParser
+from configparser import RawConfigParser
+from astropy.io import fits
+
+import numpy as np
+import sys
+
+
+
+class ConfigurationException(Exception):
+    def __init__(self,msg):
+        self.msg = msg
+        
+
+class Configuration(ArgumentParser):
+    DEFAULT       = "DEFAULT"
+    FITS          = "FITS"
+    INI           = "INI"
+    DATAIMPL      = "DATAIMPL"
+    HEADERIMPL    = "HEADERIMPL"
+    VALUE_INDEX   = 0
+    COMMENT_INDEX = 1
+
+    #--------------------------------------------------------------------------#
```

Comment:
Remove these comment lines (let's follow the common Python / Numpy / Scipy / Astropy coding style)